packages/textproc/libxml2/package.mk

```diff
@@ -21,7 +21,7 @@ PKG_CONFIGURE_OPTS_ALL="ac_cv_header_ansidecl_h=no \
              --with-zlib=$TOOLCHAIN \
              --without-lzma"
 
-PKG_CONFIGURE_OPTS_HOST="$PKG_CONFIGURE_OPTS_ALL --with-zlib=$TOOLCHAIN"
+PKG_CONFIGURE_OPTS_HOST="$PKG_CONFIGURE_OPTS_ALL --with-zlib=$TOOLCHAIN --with-python==$TOOLCHAIN"
```

Comment:
Looks a bit wrong
* "==" should "="
* really $TOOLCHAIN, and not the path to python?

Reply:
Updated using host dependancy.
checking for python... /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/toolchain/bin/python
Found Python version 3.8